lib/experimental/class-wp-theme-json-resolver-gutenberg.php

```diff
@@ -16,103 +16,6 @@
  * @access private
  */
 class WP_Theme_JSON_Resolver_Gutenberg extends WP_Theme_JSON_Resolver_6_2 {
-	/**
```

Comment:
There's something in this file that still needs to live in the experimental folder: the call to `gutenberg_add_registered_webfonts_to_theme_json`.

This is in my list of things to look at to give it some clarity and improve the developer experience. Given that the change required here is a one-liner (`get_default_block_editor_settings` to `gutenberg_get_block_theme_supports`) and we need to look at this separately, would you mind making your changes in this file directly instead of moving it to 6.2?

Reply:
Clarified in this [slack thread](https://wordpress.slack.com/archives/C02KGN5K076/p1669913300838259).